rmf_demos_fleet_adapter/rmf_demos_fleet_adapter/fleet_adapter.py

```diff
@@ -328,6 +332,12 @@ def _lane_request_cb(msg):
         state_msg.closed_lanes = closed_lanes
         closed_lanes_pub.publish(state_msg)
 
+    transient_qos = QoSProfile(
```

Comment:
You're using older alias that will throw deprecated warnings with `galactic`. Since we're moving to `humble` soon, let's stay clear of deprecations (it may not be deprecated in `humble` but to be safe)

See warnings below for newer syntax to use.

```
UserWarning: HistoryPolicy.RMW_QOS_POLICY_HISTORY_KEEP_LAST is deprecated. Use HistoryPolicy.KEEP_LAST instead.
UserWarning: ReliabilityPolicy.RMW_QOS_POLICY_RELIABILITY_RELIABLE is deprecated. Use ReliabilityPolicy.RELIABLE instead.
UserWarning: DurabilityPolicy.RMW_QOS_POLICY_DURABILITY_TRANSIENT_LOCAL is deprecated. Use DurabilityPolicy.TRANSIENT_LOCAL instead.
```